packages/core/src/actions.ts

```diff
@@ -152,11 +152,16 @@ export function toActivityDefinition<TContext, TEvent extends EventObject>(
  *
  * @param eventType The event to raise.
  */
-export function raise<TContext, TEvent extends EventObject>(
-  event: Event<TEvent>
-): RaiseAction<TEvent> | SendAction<TContext, AnyEventObject, TEvent> {
+export function raise<
+  TContext,
+  TEvent extends EventObject,
+  TSentEvent extends EventObject = AnyEventObject
```

Comment:
This shouldn't be necessary; it should be the same as `TEvent` for raising events

Reply:
`TSentEvent` ?

so like; `SendExpr<TContext, TEvent, TEvent>`, etc. instead ?

Reply:
Haven't dug that deep into the internal differences between `send` and `raise`, just assumed that since non-string events are being forwarded to `send` that using the same typings would provide the same IntelliSense & whatnot.

Reply:
> `TSentEvent` ?
> 
> so like; `SendExpr<TContext, TEvent, TEvent>`, etc. instead ?

Yes, like this, I'm pretty sure